test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
If you removed it in a branch after then its okay. It will be removed when you merge